templates/basestyle.twig

```diff
@@ -200,7 +200,7 @@
           </fieldset>
           <p>The user agent stylesheet typically has a very weird <b>&lt;fieldset&gt;</b> and <b>&lt;legend&gt;</b> styling. But hey: why not customize it to our needs?!</p>
           <fieldset>
-            <legend>Checkbox Inputs</legend>
+            <legend style="display: block;">Checkbox Inputs</legend>
```

Comment:
@MadFlip Can we avoid this? Maybe with a `br` tag?

Reply:
`br` add too big gap, so I wrapped content below legend with `div`. I did this to prevent inputs jump on the same line as `legend` title on IE